src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/ViewDeclarationOrder/Sources/ViewDeclarationOrderWithGraphInheritance.cs

```diff
@@ -0,0 +1,28 @@
+﻿using PX.Data;
+
+public class ARInvoice : IBqlTable
+{
+}
+
+public class SOInvoice : ARInvoice
+{
+}
+
+
+public class ARTran : IBqlTable { }
+
+public class SOTran : ARTran { }
+
+public class ARInvoiceEntry : PXGraph<ARInvoiceEntry, ARInvoice>
+{
+    public PXSelect<ARInvoice> ARInvoices;
+
+	public PXSelect<SOTran> SOTrans;
+}
+
+public class SOInvoiceEntry : ARInvoiceEntry
+{
+	public PXSelect<SOInvoice> SOInvoices;  // There are two cache instances: PXCache<BAccount> and PXCache<Customer>
```

Comment:
Why are you writing about `PXCache<BAccount>` and `PXCache<Customer>`?
I cannot see these caches here.

Reply:
fixed